tests/test_region_manager.py

```diff
@@ -3,6 +3,24 @@
 from sublime_lib import RegionManager, new_view, close_view
 
 from unittest import TestCase
+from unittest.mock import NonCallableMagicMock, MagicMock
+
+from inspect import signature
+
+
+class ViewMock(NonCallableMagicMock):
+    def __init__(self, callback=None):
+        super().__init__()
+
+        def side_effect(*args, **kwargs):
+            bound_args = (signature(sublime.View.add_regions)
+                          .bind(self, *args, **kwargs).arguments)
+            callback(**bound_args)
+
+        self.add_regions = MagicMock(
+            spec=sublime.View.add_regions,
+            side_effect=side_effect if callback else None
```

Comment:
The `callback` arg doesn't seem to be used?

Reply:
Whoops; that's leftover code. Removed.